src/app/clusters/barrier-control-server/cli.xml

```diff
@@ -0,0 +1,77 @@
+<?xml version="1.0"?>
```

Comment:
Do we want to include the `cli.xml`, `*-cli.c`, `*-test.[hc]`, `plugin.properties`, `callbacks.info` files in this pull?  Do we expect to be able to use them directly?

Also, a bunch of these files seem like they need license headers.

Reply:
ultimately, they'd likely power code generation.  they lacked copyright notices in @selissia's PR, but I can add 'em.

Reply:
OK.  With the license headers, we can add these, sure.  Might be worth adding some docs to the README.md files in src/app and/or src/app/docs, but might be good to land https://github.com/project-chip/connectedhomeip/pull/2006 before messing with those.

Reply:
I've decided to remove them for now